include/cutlass/epilogue/warp/tile_iterator_tensor_op_mixed.h

```diff
@@ -605,7 +605,7 @@ class TileIteratorTensorOpMixed<WarpShape_, OperatorShape_, int32_t, 32, OutputS
     TensorRef const &ref,
     unsigned lane_id
   ):
-    stride_(ref.stride()[0] / AccessType::kElements) { 
+    stride_(size_t(ref.stride()[0]) / AccessType::kElements) {
```

Comment:
`stride_` is an `int`. It's not clear to me how casting to `size_t` would fix an integer conversion warning. Is `size_t` signed on your platform?

Reply:
(See [include/cutlass/epilogue/warp/tile_iterator_tensor_op_mixed.h](https://github.com/NVIDIA/cutlass/pull/1380/files/8a136aa052c7d0f00f715bc40540b22eaca73e2a#diff-d1995fff7e8e50991dca5cbf5be60b37b52b5db3ad06ee80a83724994e08c58b))